app/views/devise/mailer/reset_password_instructions.html.erb

```diff
@@ -1,10 +1,33 @@
-<div class="container">
-    <%= render partial: 'shared/welcome', locals: { user: @user } %>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Change Password Request</title>
+  </head>
+  <body style="font-family: Arial, sans-serif; font-size: 16px; line-height: 1.4; color: #222222; background-color: #f5f5f5; margin: 0; padding: 0;">
+    <div style="max-width: 600px; margin: 0 auto;">
+      <table style="border-collapse: collapse; width: 100%;">
+        <tr>
+          <td>
+            <h1 style="font-size: 2rem; font-weight: bold;">BVCOG Contract Manager</h1>
+            <h1 style="font-size: 1.5rem; font-weight: bold;">Password Change Request</h1>
+          </td>
+        </tr>
+        <tr>
+          <td style="background-color: #ffffff; border-radius: 5px;">
+            <%= render partial: 'shared/welcome', locals: { user: @user } %>
 
-    <p>Someone has requested a link to change your password. You can do this through the link below.</p>
+            <p style="margin-bottom: 1rem;">Someone has requested a link to change your password. You can do this through the link below.</p>
```

Comment:
We don't want to be inline styling if we can avoid it. Especially stuff like margins which should remain consistent across the app. We can ignore it given the time crunch of Iteration 3 but make a note to fix it please.